clang/lib/Driver/ToolChains/Arch/RISCV.cpp

```diff
@@ -169,6 +169,9 @@ void riscv::getRISCVTargetFeatures(const Driver &D, const llvm::Triple &Triple,
     Features.push_back("-relax");
   }
 
+  if (!Args.hasArg(options::OPT_menable_experimental_extensions))
+    Features.push_back("+no-experimental-ext");
```

Comment:
We also check `OPT_menable_experimental_extensions` in `getArchFeatures` and push a different feature when it set. Do we need 2 extensions that are the opposite of each other?